bindings/python/tests/check_import.py

```diff
@@ -0,0 +1,30 @@
+
+import sys
+import importlib
+import warnings
+
+
+def check_import(module_name, package=None, skip=False, warn=None):
+    """Try to import `module_name`.
+
+    Arguments
+        module_name: Name of module to import.
+        package: Only needed for relative imports.
+        skip: Whether to skip the test if the module cannot be imported.
+        warn: Whether to warn if the module cannot be loaded.  The default
+            is to warn if `skip` is false.
+
+    Returns:
+        A module object if the module can be imported.  Otherwise None is
+        returned.
+    """
+    try:
+        module = importlib.import_module(module_name, package)
+    except ImportError:
+        if skip:
+            sys.exit(44)
```

Comment:
Why 44 error? Maybe this should be mentioned in the doc string. "Tries to import, else returns a 44 <description> error

Reply:
You are right, we should have chosen error 42 :-)

Joke aside, it is a specific error code that tells cmake to skip the test. A comment is now added